src/SvelteTable.svelte

```diff
@@ -119,7 +120,7 @@
                 on:click={() => (activeModal = col.helpModal)}
               >
                 <IconTooltip />
-                <span class="sr-only">Show tooltip</span>
+                <span class="sr-only">Show tooltip for {col.title}</span>
```

Comment:
It could be several cases:
1. there is no title, and titleComponent instead
2. the title is an empty string (example in BookingHistoryTable)